apps/cyberstorm-remix/app/commonComponents/Footer/Footer.css

```
@@ -1,251 +1,127 @@
-    flex-shrink: 0;
-    gap: var(--gap-3xl);
     justify-content: center;
-  }
 
-  .footer__icon-link {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    height: var(--space-32);
-    border-radius: var(--radius-md);
+    width: min(100%, var(--footer-inner-width));
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)